### PR TITLE
docs: restructure to create a "batch change" section

### PR DESCRIPTION
**Pull request by tianzhou**
Move the `Tenant GitOps` to the VCS section so that we can keep the `Batch change` section layer flat and reduce the content on the `Change Multiple Tenants` page.

I notice the tenant doc is out-of-date (the Label section is no longer applicable). But that's out of the scope of this PR

CleanShot 2023-02-09 at 16 56 58

### CODE REVIEW

**Comment by tianzhou** [comment]
@Candybase

File: content/docs/batch-change/multi-tenant-change.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Tenant Database Management
+title: Change Databases from Multiple Tenants
 ---
 
-Tenant Database Management allows database administrators to manage **a collection of databases with identical schemas**, these databases are often referred as [tenant databases](/docs/concepts/tenant-database).
+Bytebase allows your to change **a collection of databases with identical schemas**, these databases are often referred as [tenant databases](/docs/concepts/tenant-database).
```
Comment (boojack) [inline content/docs/batch-change/multi-tenant-change.md:5]:
```suggestion
Bytebase allows you to change **a collection of databases with identical schemas**, these databases are often referred as [tenant databases](/docs/concepts/tenant-database).
```

File: content/docs/batch-change/multi-environment-change.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Change Databases from Multiple Environments
+---
+
+A development pipeline usually consists of multiple environments such as dev, test, integration, staging, prod.
+Each environment has its own isolated database to store the respective data. A schema change will
+be propagated from each databases following the development pipeline.
```
Comment (boojack) [inline content/docs/batch-change/multi-environment-change.md:7]: Is this an unwanted line break?

```suggestion
Each environment has its own isolated database to store the respective data. A schema change will be propagated from each databases following the development pipeline.
```

File: content/docs/batch-change/overview.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Batch Change
+---
+
+Bytebase allows you to change a collection of databases in a single workflow. These databases usually
+have a homogenous structure while belong to different development environments, geographic locations, or SaaS tenants. Bytebase supports two most typical batch change scenarios:
```
Comment (boojack) [inline content/docs/batch-change/overview.md:6]:
```suggestion
Bytebase allows you to change a collection of databases in a single workflow. These databases usually have a homogenous structure while belong to different development environments, geographic locations, or SaaS tenants. Bytebase supports two most typical batch change scenarios:
```

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**